src/app/(proper_react)/DashboardTopBanner.tsx

```diff
@@ -0,0 +1,100 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styles from "./DashboardTopBanner.module.scss";
+import { ReactElement } from "react";
+import { Button } from "../components/server/Button";
+
+export type DashboardTopBannerProps = {
+  type: "LetsFixDataContent" | "DataBrokerScanUpsellContent" | "NoExposuresFoundContent" | "ResumeBreachResolutionContent" | "YourDataIsProtected";
+  chart: ReactElement,
+  data?: {
+    dataBreaches?: number,
+    infoForSale?: number,
+    exposures?: {
+      total?: number,
+      fixed?: number,
+      remaining?: number,
+    }
+  }
+}
+
+export const DashboardTopBanner = (props: DashboardTopBannerProps) => {
+  const contentData = {
+    LetsFixDataContent: {
+      headline: "Let's protect your data",
+      description: `We found your data in ${props.data?.dataBreaches} data breaches and ${props.data?.infoForSale} sites selling your personal info. We'll guide you on how to fix it.`,
```

Comment:
re: `${props.data?.dataBreaches}` does this content get suppressed if there were no breaches or data passed?
Curious if this could potentially log:
> "We found your data in _undefined_ data breaches and _undefined_ sites selling your personal info..."